src/app/components/app/app.component.ts

```diff
@@ -31,7 +31,7 @@ export class AppComponent extends ParentComponent implements OnInit {
     const user = Object.assign(new User(), JSON.parse(localStorage.getItem('currentUser')));
     if (user && user.token && user.token !== '') {
       this.store.dispatch({type: SET_USER, payload: user});
-      this.router.navigateByUrl('/channels');
+      this.router.navigateByUrl('/campaigns');
```

Comment:
This is unfortunately not correct. Default page should still be channels. David only wanted the Campaigns to be renamed to "Crowdfunding" in the header.